WordPress/Classes/ViewRelated/Stats/Helpers/SiteStatsInformation.swift

```diff
@@ -17,10 +17,6 @@ import WordPressComStatsiOS
 
     // MARK: - Instance Methods
 
-    @objc static func resetSharedInstance() {
-        sharedInstance = SiteStatsInformation()
```

Comment:
It's possible that the ship has sailed, but does this class need to be a singleton?

Reply:
Could be. Actually, `SiteStatsInformation.swift` was meant to be a temporary helper, so I didn't put much thought into it. But it can be changed, for sure.